src/Component.php

```diff
@@ -117,7 +117,11 @@ public function run(): void
                 throw new UserException('The CSV file is invalid: ' . $e->getMessage());
             }
 
-            $copyCommand = "mv " . $sourceFile->getPathname() . " " . $outputPath . "/" . $sourceFile->getBasename();
+            $copyCommand = "mv " .
```

Comment:
Takhle ne, prosím. 

> `$process = new Process(array('ls', '-lsa'));`
> -- https://symfony.com/doc/current/components/process.html#usage

Reply:
ježiš jasně, to mi uteklo. jsem to hledal a rychlej pohled do hintu mi neporadil, do dokumentace jsem nedošel. tušíš, proč ve vnitřnostech nepoužívají `escapeshellargs`?

Reply:
Netuším. https://github.com/symfony/symfony/issues/29469